geomesa-hbase/geomesa-hbase-datastore/src/main/scala/org/locationtech/geomesa/hbase/data/HBaseIndexAdapter.scala

```diff
@@ -691,7 +707,13 @@ object HBaseIndexAdapter extends LazyLogging {
 
     override def flush(): Unit = FlushWithLogging.raise(mutators)(BufferedMutatorIsFlushable.arrayIsFlushable)
 
-    override def close(): Unit = CloseWithLogging.raise(mutators)
+    override def close(): Unit = {
+      CloseWithLogging.raise(mutators)
+      CloseWithLogging.raise(pool)
```

Comment:
These should be either changed to `apply` or invoked in the same call, otherwise if the first call fails the second call won't be invoked. I'm not entirely sure the suggestion below will find the right implicits though...
```suggestion
      CloseWithLogging.raise(mutators, pool)
```

Reply:
Good catch, will fix